apps/trading/client-pages/market/trade-grid.tsx

```diff
@@ -169,6 +170,7 @@ export const TradeGrid = ({ market, pinnedAsset }: TradeGridProps) => {
             <MarketSuccessorProposalBanner marketId={market?.id} />
           </>
         )}
+        <MarketTerminationBanner />
```

Comment:
I think its a good time to refactor the proposal banners. I'm think something like

- single data provider (and single request) to get all proposals for a single market
- single wrapping <MarketProposals> component to control rendering priority of any affecting proposals

This will allow us to better handle cases when there are multiple proposals and also ensure shared and consistent styles between banners

Reply:
some refactoring has been done here